qiskit_nature/results/utils/protein_shape_file_gen.py

```diff
@@ -0,0 +1,148 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021, 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""An auxiliary class that gets the coordinates of aminoacids of a molecule
+ in ProteinFoldingResult."""
+from typing import Union, List, Optional
+import numpy as np
+
+from qiskit_nature.problems.sampling.protein_folding.peptide.peptide import Peptide
+
+
+class ProteinShapeFileGen:
+    """This class handles the creation of cartesian coordinates for
+    each aminoacid in a protein and generates a .xyz file.
+    It is used by the ProteinFoldingResult class.
+    """
+
+    coordinates = (1.0 / np.sqrt(3)) * np.array([[-1, 1, 1], [1, 1, -1], [-1, -1, -1], [1, -1, 1]])
```

Comment:
Please add some explanation to this value. Either as a comment or as a docstring.
I would also write class variables in all caps